metadata-ingestion/source_docs/ldap.md

```diff
@@ -34,6 +34,24 @@ source:
     # Options
     base_dn: "dc=example,dc=org"
 
+    # Optional attribute mapping to allow ldap config differences across orgs
+    attrs_mapping:
```

Comment:
@atulsaurav im wondering if this should really map "DataHub" concepts to specific attributes?

Like this:

urn_id: 
email
firstName
lastName
displayName

Instead of mapping LDAP -> LDAP concepts

Reply:
Yes, good idea! Let me make changes and send you an update later in the evening. Thanks!